### ISSUE TITLE
Testimonials Section

### ISSUE THREAD

**Issue by Lavanyaaa25** [Its-Aman-Yadav/Community-Site#20]
Problem:
Users may want to know about the experiences of existing community members before joining.

Solution:
Add a Testimonials Section where users can read feedback from satisfied community members.

Additional Context:
Testimonials will build trust and encourage more individuals to join the community.

**Comment by Its-Aman-Yadav** [comment]
yeah, testimonials section would be a great idea, thank you for your suggestion.

I am assigning this issue to you, please work on it @Lavanyaaa25

**Comment by Lavanyaaa25** [comment]
Sure👍

**Comment by Lavanyaaa25** [comment]
Will this be under gssoc?

**Comment by Its-Aman-Yadav** [comment]
yeah sure, adding the label

**Comment by Sandhyarani23** [comment]
Hello
I have the experience for this 
I have contributed to caseconnect 
In which lawyers and clients interacts i designed the testimonials section.
I can implement and improve that.
Please assign this to me.

**Comment by Lavanyaaa25** [comment]
> yeah sure, adding the label

can u add the label gssoc and level as it counts for leaderboard
